dev-compose/scripts/microk8s-retag-import.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#delete graphistry images from azure container registry
```

Comment:
does this file still make sense? can it instead call another so not multiple places these are maintained?

Reply:
This was a script made to retag images for micok8s when I made that on-site visit for chart deployment